packages/core/content-type-builder/admin/src/components/AttributeIcon/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Box } from '@strapi/parts';
+import {
+  Component,
+  CT,
+  Date,
+  Boolean,
+  DynamicZone,
+  Email,
+  Enumeration,
+  Json,
+  LongDescription,
+  Media,
+  Numbers,
+  Password,
+  Relation,
+  St,
+  Text,
+  Uid,
+} from '@strapi/icons';
+import { pxToRem } from '@strapi/helper-plugin';
+
+const types = {
+  biginteger: Numbers,
+  boolean: Boolean,
+  component: Component,
+  contentType: CT,
+  date: Date,
+  datetime: Date,
+  decimal: Numbers,
+  dynamiczone: DynamicZone,
+  email: Email,
+  enum: Enumeration,
+  enumeration: Enumeration,
+  file: Media,
+  files: Media,
+  float: Numbers,
+  integer: Numbers,
+  json: Json,
+  JSON: Json,
+  media: Media,
+  number: Numbers,
+  password: Password,
+  relation: Relation,
+  richtext: LongDescription,
+  singleType: St,
+  string: Text,
+  text: Text,
+  time: Date,
+  timestamp: Date,
+  uid: Uid,
+};
+
+const StyledCompo = styled(Box)`
+  width: ${pxToRem(32)};
+  height: ${pxToRem(24)};
+  box-sizing: content-box;
```

Comment:
This is already defined in the DS: https://github.com/strapi/parts/blob/cd851a7ff17be8b68734a942914c604f191238e6/packages/strapi-parts/src/ThemeProvider/ThemeProvider.js#L139

Reply:
Yes but in border-box. I specifically needed content-box here because of the padding applied to it, while keeping the same width :).

Reply:
Wooops, sorry 😓